source/platforms/GitHub.ts

```diff
@@ -227,13 +227,14 @@ export class GitHub implements Platform {
    * @param {string} newComment string value of comment
    * @returns {Promise<boolean>} success of posting comment
    */
-  async updateOrCreateComment(dangerID: string, newComment: string): Promise<boolean> {
+  async updateOrCreateComment(dangerID: string, newComment: string): Promise<string | undefined> {
```

Comment:
Sorry if obvious, but why is there an `undefined` type here instead of `null`?

Reply:
There's very little difference between undefined and null, but they are semantically different:`x = {}` and `x = {a:undefined}` respond the same to `x.a` but `{a: null}` is different. So I use null to define a conscious missing value - vs undefined which is kinda "not there", if you're doing an `if(x.a)` they're all the same, but some libs will treat null different from undefind